include/regs.h

```diff
@@ -133,12 +133,45 @@ constexpr auto BL_INDEX = 0;
 
 #endif
 
+struct CpuTestFlags {
+	bool HasCarry       = false;
+	bool IsOddParity    = false;
+	bool HasAuxiliary   = false;
+	bool IsZero         = false;
+	bool IsSignNegative = false;
+	bool HasOverflow    = false;
+	constexpr uint32_t AsValue() const;
+};
```

Comment:
We use `lower_camel_case` for struct members.

Reply:
thanks; fixed.